tests/test_google_search.py

```diff
@@ -60,6 +60,7 @@ def test_get_html(self):
 				search = GoogleSearch({"q": "Coffee", "location": "Austin,Texas"})
 				data = search.get_html()
 				self.assertGreater(len(data), 10)
+				self.assertIn("<html>", data)
```

Comment:
`<html>` tag can have attributes. Let's fix this test.

![image](https://github.com/serpapi/google-search-results-python/assets/282605/2181d041-a84b-4164-b483-4dde4d6c3625)


```suggestion
				self.assertIn("<html", data)
```